h2o-core/src/main/java/hex/ModelMetricsRegression.java

```diff
@@ -41,7 +41,11 @@ public static ModelMetricsRegression getFromDKV(Model model, Frame frame) {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     sb.append(super.toString());
-    sb.append(" mean residual deviance: " + (float)_mean_residual_deviance + "\n");
+    if(!Double.isNaN(_mean_residual_deviance)) {
```

Comment:
IMHO we should print "NA" instead of skipping the field - this is an use

Reply:
Now, it says `mean residual deviance: NaN`,  if it is not correct, I can change it to  `mean residual deviance: N/A`.